common/predictive-text/worker/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "compilerOptions": {
     "allowJs": false,
-    "module": "none",
-    "outFile": "./index.js",
+    "module": "commonjs",
```

Comment:
These options need to be left unchanged - everything for "production" must be able to compile without use of modules and/or `require` statements.

I won't complain about use of those within testing environments, though.

Reply:
Done!